packages/helpermodules/measurement_log.py

```diff
@@ -99,6 +105,11 @@ def save_log(folder):
                     'cp5': {'exported': 0, 'imported': 0},
                     'cp6': {'exported': 0, 'imported': 64}},
             'pv': {'all': {'imported': 251}, 'pv1': {'imported': 247}}}
+        },
+        "names": {
+            "counter0": "Mein EVU-Zähler",
+            "bat1": "Mein toller Speicher",
```

Comment:
So würde es zu den oberen Beispieldaten passen.
```suggestion
            "bat2": "Mein toller Speicher",
```